src/test/groovy/com/getkeepsafe/dexcount/DexMethodCountPluginSpec.groovy

```diff
@@ -41,41 +44,6 @@ final class DexMethodCountPluginSpec extends Specification {
         manifestFile.write(MANIFEST_FILE_TEXT)
     }
 
-    def 'unsupported project project'() {
```

Comment:
Removed because the AGP 4.1 implementation doesn't have this constraint, and in our current test setup that's the plugin version used in these tests.

When #304 lands, we should re-add this check for AGP 4.0 and below.